dnsaas/settings.py

```diff
@@ -137,23 +137,26 @@ def __getitem__(self, item):
 TEMPLATE_DIRS = ()
 
 INSTALLED_APPS = (
+    'django.contrib.admin',
+    'django.contrib.admindocs',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.sites',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'raven.contrib.django.raven_compat',
+
+    'ajax_select',
```

Comment:
did you consider https://github.com/yourlabs/django-autocomplete-light ? IMO it's much more user friendly than ajax select :)